docker-compose.dev.yml

```diff
@@ -18,6 +18,8 @@ services:
       EMBER_FEATURE_WORSHIP_MINISTER_MANAGEMENT_ENABLED: "true"
       EMBER_WORSHIP_DECISIONS_DATABASE_URL: "https://dev.databankerediensten.lokaalbestuur.lblod.info"
       EMBER_WORSHIP_ORGANISATIONS_DATABASE_URL: "https://dev.organisaties.lokaalbestuur.lblod.info"
+    ports:
+      - "80:80"
```

Comment:
So, maybe to remove this, but I thought it was easy. Frontend will also be available on port 80. Feel free to remove